### PR TITLE
Feature/sc 5787 create navigation top page

### PR DESCRIPTION
**Pull request by TomaszJuchimiuk**
# Description
-added new selectors and methods.

## Approval for review
- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

**Review by Grzesiek80 — APPROVED** [review]

File: page-objects/pages/NavigationTopPage.js [hunk 1]
```diff
@@ -10,11 +10,32 @@ module.exports = {
         initialsDDLogout: 'a[data-testid="logout"]',
         nameBox: '.dropdown-name',
         userIcon: '.btn-avatar > a',
+        navItemHelp: 'li.nav-item.help-drodown',
+        initialsHelpButton: '.list-group-item.list-group-item-action:first-child',
+        initialsRequestOrProblemButton: '.list-group-item.list-group-item-action:nth-child(2) .link',
```
Comment (yuliashikhareva) [inline page-objects/pages/NavigationTopPage.js:15]: please set the data-testids for all these selectors

Comment (yuliashikhareva) [reply page-objects/pages/NavigationTopPage.js:15]: in client, in extra branch with the same name as in integration tests

**Review by yuliashikhareva — CHANGES_REQUESTED** [review]

File: page-objects/pages/NavigationTopPage.js (hunk 1 above)
Comment (TomaszJuchimiuk) [reply page-objects/pages/NavigationTopPage.js:15]: I didn't find any data-testids on branch SC-4994-/-add-testids-for-integration-tests-2 run locally.

**Review by kgorner — CHANGES_REQUESTED** [review]
-Please also add full-screen mode items,  problem item (exclamation mark)
-Please rename the selectors of the help area without "initials..." - e.g. 
navItemHelp
navItemHelpHelpArea
navItemHelpWishProblem
navItemHelpContactAdmin
navItemHelpTraining
-Please rename the methods starting with "click"

**Review by Grzesiek80 — DISMISSED** [review]

File: page-objects/pages/NavigationTopPage.js (hunk 1 above)
Comment (yuliashikhareva) [reply page-objects/pages/NavigationTopPage.js]: I set some data-testids

**Pull request merged**